setup.py

```diff
@@ -41,7 +41,7 @@ def _read_reqs(relpath):
 
 
 if not PY3:
-  _INSTALL_REQUIRES += ('futures', 'enum34')
+    _INSTALL_REQUIRES += ('futures', 'enum34')
```

Comment:
Looks like a rogue change :)

Reply:
Pylama spotted it so I thought I would fix it on the way